pyproject.toml

```diff
@@ -21,7 +21,18 @@ classifiers = [
     "Typing :: Typed",
 ]
 requires-python = ">=3.11"
-dependencies = []
+dependencies = [
+    "matplotlib>= 3.3",
+    "scipy>1.5",
```

Comment:
I assume the rest of these are coming later, as they don't appear to be needed by what's there now (except for matplotlib and numpy).

Reply:
Yes, these will be used as we add in mode code.

Reply:
Yes - I will remove them for now though for correctness